docs/content/en/docs/migrate/metrics-observe/_index.md

```diff
@@ -11,3 +11,51 @@ Information that is published here has been reviewed for technical accuracy
 but the format and content is still evolving.
 We hope you will contribute your experiences
 and questions that you have.
+
+The SLIs and SLOs used for Keptn v1 quality gates can be ported to
+KLT KeptnMetrics and KeptnEvaluationDefintions.
+
+By default, KLT includes additional observability features
+that are not included in Keptn v1:
+
+* [DORA metrics](../../implementing/dora)
+* [OpenTelemetry observability](../../implementing/otel.md)
+
+Keptn v1 Quality Gates can be migrated into KLT metrics
+and KLT evaluations.
+
+> **Note**
+The full SLO capabilities
+provided by Keptn v1 such as weighting and scoring
+are currently under development for KLT.
+You can follow and participate in the design and implementation process at
+[Github epic 1785](https://github.com/keptn/lifecycle-toolkit/pull/1785).
```

Comment:
I will create a ticket to don't forget removing this piece when we'll finish the implementation :)

Reply:
Indeed!  When the implementation is complete, I'll need to rewrite this section plus we have a page about migrating SLO's to KLT where we can write up all the elegant details.